lib/vector/Vlib/color_read.c

```diff
@@ -58,12 +60,12 @@ int Vect_read_colors(const char *name, const char *mapset,
 
     if (strcmp(mapset, G_mapset()) == 0) {
         /* look for the regular color table */
-        sprintf(buf, "%s/%s", GV_DIRECTORY, name);
+        snprintf(buf, sizeof(buf), "%s/%s", GV_DIRECTORY, name);
```

Comment:
It is safe to assume there will be no truncation in this case, but make this "exclicit" to the compiler.

```suggestion
        (void)snprintf(buf, sizeof(buf), "%s/%s", GV_DIRECTORY, name);
```

Reply:
Done